airflow/api_connexion/openapi/v1.yaml

```
@@ -1937,7 +2032,12 @@ components:
           nullable: true
           description: Whether the DAG is paused.
         is_active:
-          description: Whether the DAG is currently seen by the scheduler(s).
+          description: |
+            Whether the DAG is currently seen by the scheduler(s).
+
+            *New in version 2.1.1*
+
+            `readOnly` *added in 2.2.0*
```

```suggestion
            *'readOnly' added in 2.2.0*
```
I prefer we just have italics. Please update in other places too.

---

Okay.